recipes/ffmpeg/all/conanfile.py

```diff
@@ -311,14 +314,14 @@ def requirements(self):
             self.requires("openssl/[>=1.1 <4]")
         if self.options.get_safe("with_libalsa"):
             self.requires("libalsa/1.2.10")
-        if self.options.get_safe("with_xcb") or self.options.get_safe("with_vaapi"):
+        if self.options.get_safe("with_xcb"):
             self.requires("xorg/system")
         if self.options.get_safe("with_pulse"):
             self.requires("pulseaudio/14.2")
         if self.options.get_safe("with_vaapi"):
-            self.requires("vaapi/system")
+            self.requires("libva/2.20.0")
         if self.options.get_safe("with_vdpau"):
-            self.requires("vdpau/system")
+            self.requires("libvdpau/1.5")
```

Comment:
Same for libvdpau, close to hardware, may cause incompatibility, but still looks following the same implementation as from distros:

AUR:
- https://archlinux.org/packages/extra/x86_64/libvdpau/
- https://archlinux.org/packages/extra/x86_64/libvdpau/files/
- https://gitlab.archlinux.org/archlinux/packaging/packages/libvdpau/-/blob/main/PKGBUILD?ref_type=heads

APT:
- https://git.launchpad.net/ubuntu/+source/libvdpau/tree/debian/rules?h=ubuntu/jammy